prow/plugins/override/override_test.go

```diff
@@ -345,6 +388,28 @@ func TestHandle(t *testing.T) {
 			},
 			checkComments: []string{"on behalf of " + adminUser},
 		},
+		{
```

Comment:
Lets please also test the scenarios of:
* No github apps auth, checkrun doesn't get overriden
* there is a checkrun that does not match the name in the `/override` and it does not get overriden (maybe just add to the existing testcase?)

Reply:
added the test cases